__tests__/src/integration.spec.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2017 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const toMatchImageSnapshot = require('../../src').toMatchImageSnapshot;
```

Comment:
Can you create a directory named `stubs` inside `__tests__` and place the test images there?

Reply:
`__tests__/stubs`